src/sentry/deletions/defaults/project.py

```diff
@@ -98,6 +100,13 @@ def get_child_relations(self, instance: Project) -> list[BaseRelation]:
                 {"snuba_query__subscriptions__project": instance},
             )
         )
+        relations.append(
+            ModelRelation(
+                Rule,
+                {"project_id": instance.id},
```

Comment:
Can you add this to the test for the task in https://github.com/getsentry/sentry/blob/master/tests/sentry/deletions/test_project.py ?